src/nvim/plines.c

```diff
@@ -881,6 +885,10 @@ int plines_win_col(win_T *wp, linenr_T lnum, long column)
 int plines_win_full(win_T *wp, linenr_T lnum, linenr_T *const nextp, bool *const foldedp,
                     const bool cache, const bool limit_winheight)
 {
+  if (decor_conceal_line(wp, wp->w_buffer, lnum - 1, false)) {
+    return 0;
```

Comment:
callsites still expect `foldedp` to be set (I get UBSAN errors otherwise), could  use `if (foldedp) { foldedp = *false }`

Reply:
We still need to update `nextp` as well so just moved it below the `hasFoldingWin()`.